locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -795,24 +845,48 @@ protected JcloudsSshMachineLocation obtainOnce(ConfigBag setup) throws NoMachine
 
             // Apply any optional app-specific customization.
             for (JcloudsLocationCustomizer customizer : getCustomizers(setup)) {
-                customizer.customize(this, computeService, sshMachineLocation);
+                if (windows) {
+                    LOG.warn("Ignoring customizer {} on Windows location {}", customizer, winRmMachineLocation);
```

Comment:
Tricky; we should update the customizer's interface to take a MachineLocation instead of an ssh machine location.

Let's do that in a separate PR.

Can you keep a list of what we're deferring to later PRs please.